default.nix

```diff
@@ -304,118 +368,152 @@
+          }
+        )).overrideAttrs
+          (orig: {
+            configureFlags = [
+              "--datarootdir=${placeholder "out"}/share"
+              "--disable-cs"
+              "--disable-dc"
+              "--disable-inband"
+              "--disable-openssl"
+              "--disable-rdmem"
+              "--disable-shared"
+              "--disable-xml2"
+              "--enable-adb-generic-tools"
+              "--enable-all-static"
+              "--enable-static"
+              "--enable-static-libstdcpp"
+              # "--disable-dpa" # disabling this causes it to be enabled?
+              # "--disable-fw-mgr" # disabling this causes it to be enabled?
+            ];
 
-        nativeBuildInputs = orig.nativeBuildInputs ++ [ self.nukeReferences ];
+            nativeBuildInputs = orig.nativeBuildInputs ++ [ self.nukeReferences ];
 
-        buildInputs = [ self.python3Minimal ] ++ (with self.fancy; [
-          boost
-          curl
-          expat
-          libxml2
-          openssl.out
-          xz
-          zlib
-        ]);
+            buildInputs = [
+              self.python3Minimal
+            ]
+            ++ (with self.fancy; [
+              boost
+              curl
+              expat
+              libxml2
+              openssl.out
+              xz
+              zlib
+            ]);
 
-        preFixup = (orig.postFixup or "") + (with self.fancy; ''
-          rm -f "$out/bin/mstarchive"
-          rm -f "$out/bin/mstfwmanager"
-          find "$out" \
-            -type f \
-            -exec nuke-refs \
-            -e "$out" \
-            -e ${openssl.out} \
-            -e ${self.python3Minimal} \
-            -e ${busybox} \
-            -e ${stdenv.cc.libc} \
-            '{}' +;
-        '');
-      });
+            preFixup =
+              (orig.postFixup or "")
+              + (with self.fancy; ''
+                rm -f "$out/bin/mstarchive"
+                rm -f "$out/bin/mstfwmanager"
+                find "$out" \
+                  -type f \
+                  -exec nuke-refs \
+                  -e "$out" \
+                  -e ${openssl.out} \
+                  -e ${self.python3Minimal} \
+                  -e ${busybox} \
+                  -e ${stdenv.cc.libc} \
+                  '{}' +;
+              '');
+          });
+
+      perftest = (
+        optimizedBuild (self.callPackage ./nix/perftest { inherit (versions.perftest) rev hash; })
+      );
+
+      #### NEW
+      hwloc = optimizedBuild (
+        super.hwloc.override {
+          expat = null;
+          ncurses = null;
+        }
+      );
+
+      fancy.libcap_ng = optimizedBuild super.libcap_ng;
+      fancy.libseccomp = optimizedBuild super.libseccomp;
+
+      fancy.rustPlatform = (
+        super.makeRustPlatform {
+          stdenv = fancy.stdenv;
+          cargo = super.rust-toolchain;
+          rustc = super.rust-toolchain;
+        }
+      );
+      ### /NEW
```

Comment:
The commit title says “fix: run formatters”, can we stick to running the formatters, please? The diff is large and hiding new changes in the middle is not ideal, if we need to revert a change this will make it much more complicated.

Reply:
yup.  This is just an accident.  I even marked that section as new specifically so that I wouldn't squash it into this commit and I just messed it up